src/main/java/ch/njol/skript/expressions/ExprMaxPlayers.java

```diff
@@ -18,31 +18,33 @@
  */
 package ch.njol.skript.expressions;
 
-import org.bukkit.Bukkit;
-import org.bukkit.event.Event;
-import org.bukkit.event.server.ServerListPingEvent;
-import org.eclipse.jdt.annotation.Nullable;
-
-import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
 import ch.njol.skript.Skript;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser;
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.Server;
+import org.bukkit.event.Event;
+import org.bukkit.event.server.ServerListPingEvent;
+import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Max Players")
 @Description({"The count of max players. This can be changed in a <a href='events.html#server_list_ping'>server list ping</a> event only.",
-		"'real max players' returns the real count of max players of the server always and can't be changed."})
+		"'real max players' returns the real count of max players of the server and can be modified on Paper 1.16 or later."})
 @Examples({"on server list ping:",
 		"	set the max players count to (online players count + 1)"})
-@Since("2.3")
+@RequiredPlugins("Paper 1.16+ (modify max real players)")
+@Since("2.3, INSERT VERSION (modify max real players)")
 public class ExprMaxPlayers extends SimpleExpression<Long> {
```

Comment:
This can return an Integer as both ServerListPingEvent and getMaxPlayers returns an Integer.